<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Laconnected</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/resource/1.gif</normaloff>:/resource/1.gif</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="RotatingStackedWidget" name="stackedWidget">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="logonBt">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>600</width>
         <height>200</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(25, 202, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="log_otherBt">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>440</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>帮助</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>配置</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="regBt">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>410</y>
         <width>240</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#24CBFE;</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="head_bt">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../image.qrc">
            <normaloff>:/resource/1.gif</normaloff>:/resource/1.gif</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Laconnected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="narrow_bt">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../image.qrc">
            <normaloff>:/comicon/ic_min_white.png</normaloff>:/comicon/ic_min_white.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="set_bt">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../image.qrc">
            <normaloff>:/comicon/ic_login_cfg.png</normaloff>:/comicon/ic_login_cfg.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_bt">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../image.qrc">
            <normaloff>:/comicon/ic_close.png</normaloff>:/comicon/ic_close.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>591</width>
         <height>149</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="QQlabel_4">
              <property name="minimumSize">
               <size>
                <width>112</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="stateBt_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>112</width>
                <height>20</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="focusPolicy">
               <enum>Qt::WheelFocus</enum>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
              </property>
              <item>
               <property name="text">
                <string>在线</string>
               </property>
               <property name="icon">
                <iconset resource="../image.qrc">
                 <normaloff>:/resource/在线.png</normaloff>:/resource/在线.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Q我吧</string>
               </property>
               <property name="icon">
                <iconset resource="../image.qrc">
                 <normaloff>:/resource/Q我吧.png</normaloff>:/resource/Q我吧.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>忙碌</string>
               </property>
               <property name="icon">
                <iconset resource="../image.qrc">
                 <normaloff>:/resource/忙碌.png</normaloff>:/resource/忙碌.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>请勿打扰</string>
               </property>
               <property name="icon">
                <iconset resource="../image.qrc">
                 <normaloff>:/resource/勿打扰.png</normaloff>:/resource/勿打扰.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>离开</string>
               </property>
               <property name="icon">
                <iconset resource="../image.qrc">
                 <normaloff>:/resource/离开.png</normaloff>:/resource/离开.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>隐身</string>
               </property>
               <property name="icon">
                <iconset resource="../image.qrc">
                 <normaloff>:/resource/隐身.png</normaloff>:/resource/隐身.png</iconset>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="userlabel_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>22</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="UserEdit_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>240</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="placeholderText">
                   <string>Laconnected账号/手机号/邮箱</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="locklabel_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>22</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="PasswordEdit_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>240</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <underline>false</underline>
                   </font>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>密码</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>30</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="re_password_4">
                <property name="text">
                 <string>记住密码</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="auto_log_4">
                <property name="text">
                 <string>自动登录</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>服务器设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>服务器地址配置：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="ser_setip_Edit">
                <property name="text">
                 <string>192.168.238.1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>消息服务器端口：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="ser_setprot_Edit">
                <property name="text">
                 <string>8800</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>317</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cancel_bt">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="yes_bt">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotatingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>rotatingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
